brayns/io/loaders/volume/VolumeLoader.cpp

```diff
@@ -207,50 +247,40 @@ std::vector<std::string> RawVolumeLoader::getSupportedExtensions() const
     return {"raw"};
```

Comment:
Useless braces.

Reply:
`const char *` cannot be implicitly converted into `std::vector`